src/types.ts

```diff
@@ -44,3 +44,13 @@ type SearchResponseSummary = {
   text?: string;
   status?: string;
 };
+
+export type AllowedStyleOverrides = {
+  background?: string;
+  border?: string;
+  color?: string;
+  fontFamily?: string;
+  fontSize?: string;
+  height?: string;
+  padding?: string;
+};
```

Comment:
These seemed to make the most sense to me, but we can add/remove as we see fit. What this allow-list lets us do is ensure that critical styles (i.e. the ones that maintain the general structure and positioning that we intend for the component) cannot be broken.